design/ceph/ceph-external-cluster.md

```diff
@@ -121,3 +121,39 @@ they could simply create them with unique CRD names.
 
 Assuming the object store can be shared with the external cluster, similarly to pools, the owner of the object store is the external cluster.
 If the local cluster attempts to change the pool settings such as replication, they will be ignored.
+
+## Monitoring (prometheus)
+
+Rook already creates and injects service monitoring configuration, consuming what the ceph-mgr prometheus exporter module generates.
+This enables the capability of a Kubernetes cluster to gather metrics from the external cluster and feed them in Prometheus.
+
+The idea is to allow Rook-Ceph to connect to an external ceph-mgr prometheus module exporter.
+
+1. Enhance external cluster script:
+    1. the script tries to discover the list of managers IP addresses
+    2. if provided by the user, the list of ceph-mgr IPs in the script is accepted via the new `--prometheus-exporter-endpoint` flag
+
+2. Add a new entry in the monitoring spec of the `CephCluster` CR:
+
+```go
+// MgrPrometheusExporterExternalEndpoints point to existing Ceph prometheus exporter endpoints
+MgrPrometheusExporterExternalEndpoints []v1.EndpointAddress `json:"mgrPrometheusExporterExternalEndpoints,omitempty"`
+}
+```
+
+So the CephCluster CR will look like:
+
+```yaml
+monitoring:
+  # requires Prometheus to be pre-installed
+  enabled: true
+  mgrPrometheusExporterExternalEndpoints:
+    - ip: "192.168.0.2"
+    - ip: "192.168.0.3"
```

Comment:
for the example maybe use a more random ip instead of a 192.168 prefix which is not public

Reply:
I don't think we are looking at public IP addresses here, anything on an L2 network works. That's the main assumption about the external cluster functionality, either a common L2 domain and/or proper routing.

Reply:
Not a big deal, 192.168 is just typically used for home networking.